src/vkutil.h

```diff
@@ -36,6 +36,9 @@
 #include <stdint.h>
 #include <sys/types.h>
 
+/* Static assert macro for C99. */
+#define vk_static_assert(cond) extern char vk_static_assert[sizeof(char[1 - 2 * !(cond)])]
```

Comment:
```suggestion
#define vk_static_assert(cond) extern char vk_static_assert[sizeof(char[(cond) ? 1 : -1])]
```

Reply:
Just because I'm slow and the `1 - 2 * !(cond)` took me too long to parse :smile: